scripts/spcgeonode/docker-compose.yml

```diff
@@ -153,19 +169,22 @@ services:
 
   # PostGIS database.
   postgres:
-    image: mdillon/postgis:9.6-alpine
+    image: geonode/postgis:10

```

Comment:
As this is a custom image, it would be better to have a `build` parameter.

Reply:
The current geonode/imageName setups do not reside in the main repo. I would have to merge Geonode/postgist-docker into this repo. We should discuss if this is wanted/ smart.

Reply:
Dockerfiles for spc are inside the repository; I would like to avoid duplication too, and it would make sense to me to have these file in the main repository. We should open a new issue to discuss about that I suppose.

Reply:
I have sent an email to the dev list to discuss this idea